vllm/model_executor/models/minicpmv.py

```diff
@@ -163,35 +176,44 @@ class Resampler(nn.Module):
 
     default_norm_layer = partial(nn.LayerNorm, eps=1e-6)
 
-    def __init__(self,
-                 num_queries: int,
-                 grid_size: int,
-                 embed_dim: int,
-                 num_heads: int,
-                 kv_dim: Optional[int] = None,
-                 norm_layer: nn.Module = default_norm_layer,
-                 adaptive: bool = False,
-                 max_size: Tuple[int, int] = (70, 70),
-                 version: Tuple[int, int] = (2, 0)):
+    def __init__(
+            self,
+            num_queries: Optional[int],
+            grid_size: Optional[int],
+            embed_dim: int,
+            num_heads: int,
+            kv_dim: Optional[int] = None,
+            norm_layer: Callable[[int], nn.LayerNorm] = default_norm_layer,
+            adaptive: bool = False,
+            max_size: Tuple[int, int] = (70, 70),
+            version: Tuple[int, int] = (2, 0),
+    ):
         super().__init__()
 
         self.version = version
```

Comment:
Should we refactor `Resampler` according to the version as well?

Reply:
We should. I will make the changes as soon as possible.